web/js/userTasksReport.js

```diff
@@ -527,46 +532,50 @@ Ext.onReady(function () {
     //function to show only a subset of columns and hide the others
     function showStandardView() {
         columnModel.setHidden(0, false);  //date
-        columnModel.setHidden(1, false);  //init
-        columnModel.setHidden(2, false);  //end
-        columnModel.setHidden(3, false);  //project
-        columnModel.setHidden(4, false); //task type
-        columnModel.setHidden(5, true);   //telework
-        columnModel.setHidden(6, true);   //onsite
-        columnModel.setHidden(7, true);   //story
-        columnModel.setHidden(8, false);  //taskStory
-        columnModel.setHidden(9, false);  //description
+        columnModel.setHidden(1, true);  //init
+        columnModel.setHidden(2, true);  //end
+        columnModel.setHidden(3, false);  //hours
+        columnModel.setHidden(4, false);  //project
+        columnModel.setHidden(5, false); //task type
+        columnModel.setHidden(6, true);   //telework
+        columnModel.setHidden(7, true);   //onsite
+        columnModel.setHidden(8, true);   //story
+        columnModel.setHidden(9, false);  //taskStory
+        columnModel.setHidden(10, false);  //description
         columnModel.setColumnWidth(0, 80);
         columnModel.setColumnWidth(1, 55);
         columnModel.setColumnWidth(2, 55);
-        columnModel.setColumnWidth(3, 200);
-        columnModel.setColumnWidth(4, 120);
-        columnModel.setColumnWidth(8, 120);
-        columnModel.setColumnWidth(9, 435);
+        columnModel.setColumnWidth(3, 55);
+        columnModel.setColumnWidth(4, 200);
+        columnModel.setColumnWidth(5, 120);
+        columnModel.setColumnWidth(6, 120);
+        columnModel.setColumnWidth(7, 435);
```

Comment:
Please double-check the column width values, there are some mistakes. For example, the description column used to be 435 pixels width, but now you assigned that value to the onsite column.